src/flint/FlintTypes.jl

```diff
@@ -1600,6 +1600,7 @@ mutable struct FqNmodFiniteField <: FinField
    ninv :: Int
    norm :: Int
    sparse_modulus :: Int
+   is_conway::Int
```

Comment:
```suggestion
   is_conway::Cint
```

Reply:
Thanks, you are correct. I will fix this.